echopype/calibrate/calibrate_ek.py

```diff
@@ -608,8 +611,8 @@ def _get_gain_for_complex(self, waveform_mode, freq_center) -> xr.DataArray:
             ``CW`` for CW-mode samples, either recorded as complex or power samples
             ``BB`` for BB-mode samples, recorded as complex samples
         freq_center : xr.DataArray
-            Nominal channel frequency for CW mode samples
-            and an xr.DataArray with coorindate ``frequency`` and ``ping_time`` for BB mode samples
+            Nominal channel for CW mode samples
+            and an xr.DataArray with coorindate ``channel`` for BB mode samples
```

Comment:
Seems that `freq_center` here can be changed to `chan_sel` so that you pass in the channel id directly, since all the use of `freq_center` within this function is `freq_center.channel`.

Reply:
also so in `_cal_complex` there's a more consistent pattern to pass in the channel, right now it is
```
gain = self._get_gain_for_complex(waveform_mode=waveform_mode, freq_center=chan_sel)
```